.github/workflows/ci.yml

```diff
@@ -5,7 +5,7 @@ on:
     paths-ignore:
       - 'docs/**'
       - '*.md'
-  pull_request_target:
+  pull_request:
```

Comment:
We should use `pull_request_target`. It was added to avoid exposing secrets to forks:

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target
```
This event runs in the context of the base of the pull request, rather than in the context of the merge commit, as the pull_request event does. This prevents execution of unsafe code from the head of the pull request that could alter your repository or steal any secrets you use in your workflow. This event allows your workflow to do things like label or comment on pull requests from forks. Avoid using this event if you need to build or run code from the pull request.
```

Reply:
That's broken. It's running the code from the main branch and not the target branch.

Reply:
Right. The "base branch" is `main`. That's exactly what it is supposed to do.

Reply:
The full code that it runs is main, there is nothing of the current PR that is checkout or run. The code and workflow is now identical to main so it's kind of useless for this specific job where we need the current code.

Reply:
Check out https://github.com/fastify/fastify/pull/3737, apparently the updates to package.json and config files where not picked up.